### ISSUE TITLE
Export task perimeter as GPX

### ISSUE THREAD

**Issue by nlehuby** [hauke96/simple-task-manager#228]
Would it be possible to add a button on the task panel to export the perimeter as GPX ?

Here is my use case : I want to use SimpleTaskManager to map an area but I need to do some field survey before I can start mapping.
My plan is to organize the survey too with SimpleTaskManager : pick up a task, go outside to survey the task area, and then add a few process point when I'm done.
I need the geometry of the task to make sure I'm only surveying my task. GPX would be the most usable I think (works great with Organic Maps and OSMAnd for instance)

For now, I use the Open in JOSM button, save the perimeter as GPX, add back the number of the task as the name of the track, then open it in my mobile app. This is quite tedious...

**Comment by hauke96** [comment]
Hi, thanks for the feature request. I can definitely see the use case but this is a quite specific feature. Maybe it can be generalized into the following:

There is already an export functionality in the settings area. However, it's a custom JSON schema acting as a backup of the whole project, so not really usable in GIS tools/apps. Offering other formats as well (GeoJSON, GPX with routes, GPX with tracks, ...) might be nice for other people as well to load the tasks into all kinds of GIS tools and apps.

I think this would also cover your use case, wouldn't it?

**Comment by nlehuby** [comment]
yes, sure, as long as you can get the geometry and the task number.

**Comment by hauke96** [comment]
Perfect, then I'll do something like that :) Can't promise any results soon though.
